harness/determined/cli/trial.py

```diff
@@ -120,6 +125,71 @@ def trial_logs(args: Namespace) -> None:
         timestamp_after=args.timestamp_after,
     )
 
+@authentication.required
+def output_logs(args: Namespace): 
+    if args.output_dir is None: 
+        raise Exception('Please provide the output directory you would like to direct the logs to.')
+    
+    output_dir_tar = tarfile.open(f'{args.output_dir}.tar.gz', 'w:gz')
```

Comment:
We should, by default, write to PWD and it should be something like:

```
if output_dir is None:
    output_dir = os.getcwd()
filename = "det-bundle-trail<trial_num>-<datetime>.tar.gz"
fullpath = os.path.join(output_dir, filename)
bundle = tarfile.open(fullpath, "w:gz")
...
```

Maybe use Temp Dirs and/or Temp Files so that they get auto-cleaned up, like this:
```
with tempfile.TemporaryDirectory() as temp_dir:
    trial_logs_filepath = write_trial_logs(args, temp_dir)
    bundle.add(trial_logs_filepath, arcname=os.path.basename(trial_logs_filepath))
    ...
```

I didn't test the above but you get the idea.

Reply:
Yes this definitely makes things a lot cleaner! Thank you.